### DISCUSSION TITLE
New version found

### DISCUSSION THREAD

**Discussion by Reencito** [farag2/Sophia-Script-for-Windows#465] [Q&A] [answered]
When I try to run the script it says this to me and immediately brings me here, I installed the new script version but that doesnt work with my windows version so, how can i execute the script?
![image](https://user-images.githubusercontent.com/128880043/227695728-99743ac9-0a60-4f02-ae72-2c7d61167e0d.png)
It means 'A new version found'

**Comment by farag2** [comment]
No way to do that. You have to use the new version with bug fixed.

**Comment by farag2** [comment, marked as answer]
https://github.com/farag2/Sophia-Script-for-Windows/releases/tag/6.4.2
